src/Data/ExercisesData.js

```diff
@@ -0,0 +1,54 @@
+//This function holds some data for the mock exercises we have currently
+
+export default function ExercisesData() {
+      
+    return [{    
+        exerciseName: "Jumping Jacks",
+        reps: "30 reps",
+        id: "Jumping Jacks",
```

Comment:
Ther is no indentation

Reply:
this is the json "like" file for the mock-up data, there is no indentation on the key-value pairs

Reply:
@Darango84 When I use format document in VS code it has some indentation